runners/flink/1.12/build.gradle

```diff
@@ -22,8 +22,8 @@ project.ext {
   // Set the version of all Flink-related dependencies here.
   flink_version = '1.12.7'
   // Version specific code overrides.
-  main_source_overrides = ['./src/main/java']
-  test_source_overrides = ['./src/test/java']
+  main_source_overrides = ["${basePath}/1.11/src/main/java", './src/main/java']
```

Comment:
Flink 1.11 has been deleted from Beam. 1.12 is the current minimum version.

I realized I accidentally left a file in the Flink 1.11 directory, sent a PR to fix: #17750

Reply:
Ah yes I did not realize that :) Thanks for the quick response. I fixed the PR accordingly.